### PR TITLE
Pr/fix init error

### PR DESCRIPTION
**Pull request by natecavanaugh**
When findUp can't find an existing package-scripts.js file, it returns null, which is passed back to
findUp and throws an error because path.resolve can only accept strings as arguments

#124

**What**: Fixes error thrown when a package-scripts.js file doesn't exist while running `init`

**Why**: Because if you're running `nps init` without an existing package-scripts.js file, it won't work, forcing developers to have to create the file manually.

**How**: Checking that the `path` returned from `getPSConfigFilepath` is actually a string before passing it to `findUp.sync`.

I tried figuring out why the tests never fail for this, but I believe it's because of jest's mocking of findUp means that it never even hits the `path.resolve` method, so I'm not sure how you'd want to test for this particular case.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -266,6 +266,13 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "natecavanaugh",
+      "name": "Nate Cavanaugh",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/116871?v=3",
+      "profile": "http://alterform.com",
+      "contributions": []
```
Comment (kentcdodds) [inline .all-contributorsrc]: Could you add `"code"` to this array and then run `npm start generateContributors`?

**Comment by kentcdodds** [comment]
This is great, thank you!

File: .all-contributorsrc (hunk 1 above)
Comment (natecavanaugh) [inline .all-contributorsrc]: Done, thanks for the heads up :)

**Comment by kentcdodds** [comment]
Thank you so much!

**Pull request merged**